libraries/AP_Scheduler/AP_Scheduler.h

```diff
@@ -87,6 +97,13 @@ class AP_Scheduler
         RECORD_TASK_INFO = 1 << 0
     };
 
+    enum FastTaskPriorities {
+        FAST_TASK_PRI0 = 0,
+        FAST_TASK_PRI1 = 1,
+        FAST_TASK_PRI2 = 2,
+        MAX_FAST_TASK_PRIORITIES = 3
+    };
+
```

Comment:
Not used....

Reply:
Used in the macro and to check the priorities